### PR TITLE
Solution1

### PR DESCRIPTION
**Pull request by Sandzhlavi**

### CODE REVIEW

**Review by djtyrf312 — CHANGES_REQUESTED** [review]
too early for this task. Wait till the start of the cypress lessons, check this task again and resend on review

**Review by Sandzhlavi — COMMENTED** [review]
Solution

File: cypress/e2e/signIn.cy.js [hunk 1]
```diff
@@ -4,4 +4,42 @@ describe('Sign In page', () => {
...
   });
+  it('should open the "Sign In" page', () => {
+    cy.visit('https://conduit.mate.academy/user/login');
+
+    cy.url().should('include', '/login');
+
+    cy.get('input[type="email"]').should('be.visible');
+    cy.get('input[type="password"]').should('be.visible');
+  });
+  it('should fill in the email and password fields', () => {
```
Comment (KarolVlasova) [inline cypress/e2e/signIn.cy.js:15]: this is unnecessary test

**Review by KarolVlasova — CHANGES_REQUESTED** [review]
leave only 2 tests - 1 is for UI of the login form, 2 is for log in

**Pull request closed without merging**